compute_endpoint/tests/unit/test_engines.py

```diff
@@ -96,30 +74,21 @@ def test_result_message_packing():
     assert serializer.deserialize(unpacked.data) == result
 
 
-# Skipping "gc_engine" since parsl.htex.submit has an
-# unreliable serialization method.
-# @pytest.mark.parametrize("x", ["gc_engine"])
-@pytest.mark.parametrize("x", ["proc_pool_engine"])
-def test_engine_submit(x, gc_engine, proc_pool_engine):
-    "Test engine.submit with multiple engines"
-    if x == "gc_engine":
-        engine = gc_engine
-    else:
-        engine = proc_pool_engine
+@pytest.mark.skip(
+    "Skipping gc_engine since parsl.htex.submit has an unreliable serialization method."
+)
```

Comment:
Is this skip message still true?  Is that at all concerning for our robustness guarantees?  Seems like this might be something we very _much_ want to specifically test and verify.

And if we can then identify the core issue, we can take a fix back to Parsl.

Reply:
This was fixed. Removing the skip